pkg/testgridanalysis/testgridconversion/synthentic_tests.go

```diff
@@ -42,31 +44,67 @@ func createSyntheticTests(rawJobResults testgridanalysisapi.RawData) {
 				}
 			}
 			setupFailed := jrr.SetupStatus != testgridanalysisapi.Success
+			setupSucceeded := jrr.SetupStatus == testgridanalysisapi.Success
 
-			if installFailed {
```

Comment:
reasoning through a total runs difference.  Previously, to be counted as install success or failure you had to 

1. have some operator results, with at least one failed, OR
2. have completed the setup container successfully.  There was a bug that prevented the setup container from being found on platforms without setup containers


In the new world
1. setupsucceeded, with the detection bug fixed
2. have some operator results.

To my mind, this means we need to have a way to detect when we aren't finding a setup container.